pvlib/pvsystem.py

```diff
@@ -811,6 +812,10 @@ def physicaliam(aoi, n=1.526, K=4., L=0.002):
     spa
     ashraeiam
     '''
+    zeroang = 1e-06
+
+    aoi = np.where(aoi == 0, zeroang, aoi)
```

Comment:
I am surprised that the model would not work for an angle of exactly zero degrees.  Could this have something to do with the equation discrepancy mentioned in the comments further above?